### PR TITLE
optimize returned data while fetching

### PR DESCRIPTION
**Pull request by trikimahdi**

### CODE REVIEW

**Review by souissimai — APPROVED** [review]
Code ok Test ok

File: src/components/dialogs/network-modifications/line-to-attach-or-split-form/line-to-attach-or-split-form.js [hunk 1]
```diff
@@ -13,27 +13,43 @@ import { areIdsEqual, getObjectId } from 'components/utils/utils';
...
+    });
 
     useEffect(() => {
         if (studyUuid && currentNode?.id) {
-            fetchLines(studyUuid, currentNode.id, []).then((values) => {
-                setLinesOptions(
-                    values.sort((a, b) => a?.id?.localeCompare(b?.id))
-                );
-            });
+            console.log('inside');
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/network-modifications/line-to-attach-or-split-form/line-to-attach-or-split-form.js:32]: remove console.log

File: src/components/dialogs/network-modifications/line-to-attach-or-split-form/line-to-attach-or-split-form.js [hunk 2]
```diff
@@ -13,27 +13,43 @@ import { areIdsEqual, getObjectId } from 'components/utils/utils';
...
+            fetchEquipmentsIds(
+                studyUuid,
+                currentNode?.id,
+                [],
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/network-modifications/line-to-attach-or-split-form/line-to-attach-or-split-form.js:36]: why empty array ? i think undefined

Comment (trikimahdi) [reply src/components/dialogs/network-modifications/line-to-attach-or-split-form/line-to-attach-or-split-form.js:36]: done

**Review by Walid-Sahnoun — APPROVED** [review]
code ok
tests ok

**Pull request merged**